components/local-app/cmd/workspace-stop.go

```diff
@@ -34,54 +34,60 @@ var workspaceStopCommand = &cobra.Command{
 			return err
 		}
 
-		slog.Info("stopping workspace...")
+		slog.Debug("stopping workspace...")
 		wsInfo, err := gitpod.Workspaces.StopWorkspace(ctx, connect.NewRequest(&v1.StopWorkspaceRequest{WorkspaceId: workspaceID}))
 		if err != nil {
 			return err
 		}
 
-		currentPhase := wsInfo.Msg.GetResult().Status.Instance.Status.Phase
-
-		if currentPhase == v1.WorkspaceInstanceStatus_PHASE_STOPPED {
+		wsPhase := wsInfo.Msg.GetResult().Status.Instance.Status.Phase
+		switch wsPhase {
+		case v1.WorkspaceInstanceStatus_PHASE_STOPPED:
 			slog.Info("workspace is already stopped")
-			return nil
-		}
-		if currentPhase == v1.WorkspaceInstanceStatus_PHASE_STOPPING {
+		case v1.WorkspaceInstanceStatus_PHASE_STOPPING:
 			slog.Info("workspace is already stopping")
-			return nil
 		}
+
 		if stopDontWait {
 			slog.Info("workspace stopping")
 			return nil
 		}
 
 		stream, err := gitpod.Workspaces.StreamWorkspaceStatus(ctx, connect.NewRequest(&v1.StreamWorkspaceStatusRequest{WorkspaceId: workspaceID}))
-
 		if err != nil {
 			return err
 		}
```

Comment:
In Golang used to `defer` close / release something once it create successfully (maybe there will have some special cases but it's a good habit to do so)